go.mod

```diff
@@ -3,12 +3,10 @@ module github.com/openshift/client-go
 go 1.13
 
 require (
-	github.com/openshift/api v0.0.0-20200116145750-0e2ff1e215dd
+	github.com/openshift/api v0.0.0-20200311151921-fdf269f98861
```

Comment:
hash looks good, corresponds to https://github.com/openshift/api/tree/prebase-1.18.beta2

Reply:
Yup, the previous did not.